### PR TITLE
Adding manor.hackclub.com

### PR DESCRIPTION
**Pull request by WolfGamer2**
# Adding manor.hackclub.com

## Description

Halloween is coming, and so is The Spooky Boba Manor; this event will start in October and will be for Hack Clubers to make a manor!

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1841,6 +1842,10 @@ mangu:
   - ttl: 600
     type: CNAME
     value: mangu.pages.dev.
+manor:
+  - ttl: 600
+    type: CNAME
+    value: boba-manor.vercel.app/.
```
Comment (alx-alexpark) [inline hackclub.com.yaml:1848]:
```suggestion
    value: cname.vercel-dns.com.
```

**Review by alx-alexpark — CHANGES_REQUESTED** [review]
What a scrumptious sounding event, boba is yum! Just a few small changes and you'll be on your way!

**Comment by alx-alexpark** [comment]
And one last thing, can you confirm that this is an official HQ sponsored event?

**Comment by WolfGamer2** [comment]
Yes, this is an official hq event run by the Boba Drops team.

**Comment by maxwofford** [comment]
Confirming! I'm working with @WolfGamer2 on this

**Review by Muirrum — APPROVED** [review]
lgtm!

**Pull request merged**